client/ayon_deadline/plugins/publish/global/collect_deadline_server_from_instance.py

```diff
@@ -30,13 +31,14 @@ class CollectDeadlineServerFromInstance(pyblish.api.InstancePlugin):
     # Run before collect_render.
     order = pyblish.api.CollectorOrder + 0.225
     label = "Deadline Webservice from the Instance"
-    targets = ["local"]
+    targets = ["local", "deadline"]
 
     families = FARM_FAMILIES
 
     def process(self, instance):
-        if not instance.data.get("farm"):
-            self.log.debug("Should not be processed on farm, skipping.")
+        if (not instance.data.get("farm") and
```

Comment:
How this could happen? Validation of `"farm"` is at lot of core's functionality so this might fix it here but will break there.